### PR TITLE
Basic turbulence module

### PR DESCRIPTION
**Pull request by orlox**
Add MLT+mixing

### CODE REVIEW

File: src/StellarModels/StellarModel.jl [hunk 1]
```diff
@@ -134,11 +134,88 @@ function StellarModel(var_names::Vector{Symbol},
...
-                      eos=eos, opacity=opacity, network=network,
+                      eos=eos, opacity=opacity, network=network, turbulence=turbulence,
                       start_step_props=start_step_props, prv_step_props=prv_step_props, props=props,
                       opt=opt, plt=plt,
                       history_file=HDF5.File(-1,""),
                       profiles_file=HDF5.File(-1,""))
     return sm
 end
 
+"""
```
Comment (matthiasfabry) [inline src/StellarModels/StellarModel.jl:145]: This is deprecated

Comment (orlox) [reply src/StellarModels/StellarModel.jl:145]: what is deprecated?

Comment (matthiasfabry) [reply src/StellarModels/StellarModel.jl:145]: this whole function, there is no `ssi` anymore

**Comment by matthiasfabry** [comment]
The Docs fail is OK, I can fix that in #79.

**Comment by orlox** [comment]
You mean you would merge this with the CI failure and fix it by merging your other PR?

**Comment by matthiasfabry** [comment]
Indeed, otherwise you can adapt the `docs/` files here already, so as to keep `main` clean

File: src/StellarModels/StellarModel.jl (hunk 1 above)
Comment (orlox) [reply src/StellarModels/StellarModel.jl:145]: Don't get you, this is the StellarModel constructor

Comment (orlox) [reply src/StellarModels/StellarModel.jl:145]: Ahhh, my bad, it's what's down. Yeah, that somehow revived when I merged from main. It should die

**Pull request merged**